client/reader/tag-stream/style.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 .tag-stream__header .follow-button {
-	z-index: 1;
+	z-index: z-index( '.tag-stream__header-follow', 'button.button.follow-button.has-icon..is-following' );
```

Comment:
Not sure why there are two periods here? @scottsweb

Reply:
`has-icon..is-following`

Reply:
I used the Chrome extension to generate these and this is what it gave me